docs/use-cases/airdrop.mdx

```diff
@@ -0,0 +1,48 @@
+---
+<!-- TODO: description --->
+---
+import OutLink from "@site/src/components/OutLink";
+import IdealImage from '@site/src/components/IdealImage';
+
+# Passport-enabled airdrops
+
+Token airdrops are popular "events" on new blockchains or products. It allows making an initial list of stakeholders and "introducing" the project to the public.
+But it comes with some limitations.
+
+## Problem statement
+
+If the only airdrop requirement would be to press a "Participate" button — a lot of people would abuse it, to potentially get rich in the future by using multiple accounts.
+That's why most airdrops require some sort of KYC procedure to protect themselves.
```

Comment:
This is not KYC. It's anti-bot and anti-Sybil protection with optional filters by residency/age